src/promotions/domain/black-friday-promotion.php

```diff
@@ -14,7 +14,7 @@ class Black_Friday_Promotion extends Abstract_Promotion implements Promotion_Int
 	 */
 	public function __construct() {
 		parent::__construct(
-			'black_friday_2023',
+			'black_friday_2023_sale',
```

Comment:
Same as above, maybe use promotion and not sale. And lets reconsider not using the -

Reply:
Done 👍